src/heuristics.c

```diff
@@ -112,7 +112,9 @@ static void boot_file_heuristics(struct file *file)
 	if ((strncmp(file->filename, "/usr/lib/gummiboot", 18) == 0) ||
 	    (strncmp(file->filename, "/usr/bin/gummiboot", 18) == 0) ||
 	    (strncmp(file->filename, "/usr/bin/bootctl", 16) == 0) ||
-	    (strncmp(file->filename, "/usr/lib/systemd/boot", 21) == 0)) {
+	    (strncmp(file->filename, "/usr/lib/systemd/boot", 21) == 0) ||
+	    (strcmp(file->filename, "/usr/bin/clr-boot-manager") == 0) ||
+	    (strcmp(file->filename, "/usr/share/syslinux/ldlinux.c32") == 0)) {
 		file->is_boot = 1;
```

Comment:
Did you check if there's any implication on setting is_boot here for this file?

Reply:
hmm no 🤔 , how do I find that out? I've no idea how to research that

Reply:
Grep is_boot in the code and check everywhere that this is used. There's a code that is setting do_not_update if is_boot is 1. I don't know how this will affect the update process. I have the feeling that by adding that we are going to stop updating clr-boot-manager.